integration/spark/src/test/java/openlineage/spark/agent/lifecycle/StaticExecutionContextFactory.java

```diff
@@ -10,6 +10,7 @@
 import java.util.concurrent.TimeoutException;
 import openlineage.spark.agent.OpenLineageContext;
 import openlineage.spark.agent.OpenLineageSparkListener;
+import openlineage.spark.agent.lifecycle.plan.CommonDatasetVisitors;
```

Comment:
This doesn't really warrant its own class. I get the desire to reduce duplication, but it's really just a list of common visitors. Why not just construct the list and pass it to each of the input and output visitors?